.pyspelling-ignore-words

```diff
@@ -724,6 +725,7 @@ mkdir
 mkdocs
 mlflow
 mlmd
+mlops
```

Comment:
Good word to ignore :-D But I guess we go with MLOps as the way to write it?

Reply:
The problem is that we're referencing the repository, and I haven't figured out how to get the PySpelling filter to work to ignore URLs, so I think for now at least we have to add it.